Cargo.toml

```diff
@@ -3,8 +3,16 @@ name = "openzwave-adapter"
 version = "0.1.0"
 authors = ["Julien Wajsberg <felash@gmail.com>"]
 
+[features]
+mock = []
+
 [dependencies]
 openzwave-stateful = { git = "https://github.com/fxbox/openzwave-stateful-rust" }
 foxbox_taxonomy = { git = "https://github.com/fxbox/taxonomy.git", rev = "c222794d238072920857fea55e55d522ae75c192" }
 transformable_channels = "^0.1"
 log = "^0.3"
+taxonomy_mock = { path = "./taxonomy_mock/" }
```

Comment:
This would be better put in dev-dependencies. But dev-dependencies [can't be optional](http://doc.crates.io/manifest.html#the-features-section)